FAQ/README.md

```diff
@@ -43,3 +43,27 @@ The single-sig private key and seed phrase of existing addresses are single-sig
 ### Can we bond RUNE/ provide liquidity on THORChain by using Vultisig and will it be counted for airdrop?
 
 Yes, Vultisig will support this. It will be counted to airdrop value.&#x20;
+
+### What is the best practice in the event of losing 67% of vault shares at the same time? For example, a lady has her handbag snatched with two mobile phones inside the handbag and she has Vultisig 2of2 or 2of3 vault setup with both mobile phones.
```

Comment:
Maybe the title could be a bit more formal?

What to Do When Two Mobile Phones with 67% of Vault Shares Are Lost or Stolen: Best Practices for Vultisig 2of2 or 2of3 Vault Setups